core/src/main/resources/application.conf

```diff
@@ -118,6 +120,11 @@ call-caching {
   // request to an external service (DockerHub, GCR).  If a call fails to lookup a
   // Docker hash, it will fail.
   lookup-docker-hash = false
+
+  // This option determines whether file contents are hashed to determine cache hit or miss.
+  hash-file-contents = true
+  // This option determines whether file paths are hashed to determine cache hit or miss.
+  hash-file-paths = false
```

Comment:
Would it ever make sense to have both Docker hashing modes or file hashing modes turned on?  I wonder if these would make more sense as enumerated values:

```
call-caching {
  docker-hashing = "name"  # Could also be 'off' or 'lookup'
  file-hashing = "path" # Could also be 'off' or 'lookup'
}
```